client/src/pages/WelcomeScreen/Welcome.tsx

```diff
@@ -0,0 +1,26 @@
+import { useState } from 'react'
+import styles from './Welcome.module.css'
+import { SelectLanguage, AllowPi, FingerPrint } from '../../components'
+import { slideUp } from '../../animations';
+import { motion } from "framer-motion"
+
+export const WelcomeScreen = () => {
+  const [closeFingerPrint, setCloseFingerPrint] = useState(false)
+  return (
+    <motion.div
+      id={styles.wrapper}
+      // animate={slideLeft}
+      initial={{ x: '100%', opacity: 0 }}
+      animate={{ x: 0, opacity: 1 }}
+      exit={{ x: '-100%', opacity: 0 }}
+    >
+      <SelectLanguage />

```

Comment:
Just remove this compnent from here. We are not supporting multiple languages at the moment.

Don't delete the component cos we will use it late. Just remove it from the screen

Reply:
ok